pkg/agent/openflow/pipeline.go

```diff
@@ -2629,10 +2732,10 @@ func (f *featureMulticast) externalMulticastReceiverFlow() binding.Flow {
 	return MulticastRoutingTable.ofTable.BuildFlow(priorityLow).
 		Cookie(f.cookieAllocator.Request(f.category).Raw()).
 		MatchProtocol(binding.ProtocolIP).
-		MatchDstIPNet(*mcastCIDR).
+		MatchDstIPNet(*types.McastCIDR).
 		Action().LoadRegMark(OFPortFoundRegMark).
 		Action().LoadToRegField(TargetOFPortField, config.HostGatewayOFPort).
-		Action().NextTable().
+		Action().GotoStage(stageOutput).
```

Comment:
why this change? because there are ingress tables in-between?

Reply:
Yes, it is not necessary to goto the tables. Do you think NextTable is enough?

Reply:
as there is already a flow for IGMP query in this table.